composer/trainer/devices/device_gpu.py

```diff
@@ -33,14 +34,21 @@ def tensor_to_device(self, tensor: Tensor) -> Tensor:
     @contextmanager
     def precision_context(self, precision: Union[str, Precision]) -> Generator[None, None, None]:
         precision = Precision(precision)
+        enabled = False
         if precision == Precision.FP32:
             enabled = False
         elif precision == Precision.AMP:
             enabled = True
+        elif precision == Precision.BF16 and version.parse(torch.__version__) >= version.parse("1.10"):
```

Comment:
maybe something like:
```
elif precision == Precision.BF16:
    if version.parse(torch.__version__) < version.parse("1.10"):
        raise ValueError("BF16 precision requires torch > 1.10, got version {torch.__version__})
    enabled=True
```

Otherwise, users on torch 1.9 would get a "BF16 not supported for a GPU" error which isn't actually true.